docker-compose.dev.yml

```diff
@@ -191,18 +191,31 @@ services:
     ports:
       - '5024:5024'
 
-  # tinystories_dataset_api:
-  #   container_name: tinystories-dataset-api
-  #   image: aignostic/dev
-  #   volumes:
-  #     - .:/app
-  #     - venv:/app/.venv
-  #   command: poetry run python3 -u -m mocks.dataset.tinystories_dataset_server --host 0.0.0.0 --reload
-  #   working_dir: /app/packages/mocks
-  #   networks:
-  #     - aignostic_network
-  #   ports:
-  #     - '5026:5026'
+  tinystories_dataset_api:
+    container_name: tinystories-dataset-api
+    image: aignostic/dev
+    volumes:
+      - .:/app
+      - venv:/app/.venv
+    command: poetry run python3 -u -m mocks.dataset.tinystories_dataset_server --host 0.0.0.0 --reload
+    working_dir: /app/packages/mocks
+    networks:
+      - aignostic_network
+    ports:
+      - '5026:5026'
+
+  boston_housing_data_api:
```

Comment:
Do we need to deploy this to prod? If so please add to swarm/mocks.(rest of file name here).yml follow the example